### PR TITLE
FSA22V2-285: Refactor: remove sparkbox logo

### PR DESCRIPTION
**Pull request by marissahuysentruyt**
### Description:

-Remove branding and replace with a text header instead. 
-Add new styles for new header.

### Spec:

See Story: [FSA22V2-285](https://sparkbox.atlassian.net/browse/FSA22V2-285)

### Validation:

- [x] This PR has code changes, and our linters still pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. Run `npm install` to make sure you have all proper dependencies. (this project requires Node 16+)
4. Copy the `.env.example` file, making sure it's in the root of your project, and rename it `.env.local`. Search for "Accessible Components" in 1Password to find the API Key and Base ID. Add both of those secrets to your newly created `.env.local` file. That should establish your connection to our Airtable database.
5. Run `npm run lint` to confirm no errors.
6. Run `npm run test` to confirm all tests pass.
7. In terminal: `npm run dev`.
8. Check [localhost:3000](http://localhost:3000/) to see changes.
9. Run axe Devtools on affected pages to confirm no urgent errors.

#### Additional Valdiation
1. The header no longer contains the Sparkbox logo, and now a new link `Accessible Components` navigates to the home page. 
2. The new text link works in both dark and light modes.

### CODE REVIEW

**Review by tisidro — APPROVED** [review]
I ran axe on it and got no errors pertaining to the Accessible Component links. It displays and functions as it should. Good work! :)

**Comment by tisidro** [comment]
One quick thing I remembered since I'm working on optimizing images--you may want to remove the logo image from the public file if you didn't already :)

**Review by tisidro — DISMISSED** [review]
Looks great -- svg image removed/ I approved! :)

File: src/styles/components/_header.scss [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-      border: 2px solid var(--accent-color);
+    &:hover,
+    &:focus {
+      border-bottom: 2px solid var(--accent-color);
     }
   }
 
   @media (min-width: $breakpoint-large-screen-860) {
-    padding: 1rem 1.25rem 0 8.5625rem; // top: 16px/16; right: 20px/16; bottom: 0; left: 137px/16
+    padding: 1rem 1.25rem 1rem 8.5625rem; // top: 16px/16; right: 20px/16; bottom: 0; left: 137px/16
```
Comment (mcksheridan) [inline src/styles/components/_header.scss:24]: You'll want to update the comment on your side to reflect the fact that the bottom is no longer 0.

**Review by mcksheridan — CHANGES_REQUESTED** [review]
Looks good to me! I'm leaving one comment requesting a very minor change.

File: src/styles/components/_header.scss (hunk 1 above)
Comment (marissahuysentruyt) [reply src/styles/components/_header.scss]: Oh gosh, yup. Totally forgot about the comment. Should be fixed now: https://github.com/sparkbox/accessible-components/pull/86/commits/9e83a757ede7368c137fb626d1116e19d8547673

File: src/styles/components/_header.scss [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
-      border: 2px solid var(--accent-color);
+    &:hover,
+    &:focus {
+      border-bottom: 2px solid var(--accent-color);
     }
   }
 
   @media (min-width: $breakpoint-large-screen-860) {
-    padding: 1rem 1.25rem 0 8.5625rem; // top: 16px/16; right: 20px/16; bottom: 0; left: 137px/16
+    padding: 1rem 1.25rem 1rem 8.5625rem; // top: 16px/16; right: 20px/16; bottom: 16px/16; left: 137px/16
```
Comment (mcksheridan) [inline src/styles/components/_header.scss:24]: 👍

**Review by mcksheridan — APPROVED** [review]

**Pull request merged**